java/arcs/android/sdk/host/ArcHostHelper.kt

```diff
@@ -107,6 +109,12 @@ class ArcHostHelper(
                     intent,
                     arcHost::registeredParticles
                 )
+            Operation.LookupArcStatus ->
+                runWithResult(
```

Comment:
Nit: You can put runWithResult up on the previous line. Same for the case immediately above it.